### ISSUE TITLE
Should be able to turn off the Account Settings scrolling

### ISSUE THREAD

**Issue by pauby** [wavebox/waveboxapp#1085]
* Wavebox Version: 4.11.2
* Operating System & Version: Windows 10

**Describe the solution you'd like**
When you go into the Settings for an account / service, Wavebox scrolls to the account / service you want to manage. It's cute and fun, but it's irritating (and slow) after a while and provides no actual benefit. I would prefer an option to turn this off so that when I open the settings of an account / service it opens exactly to it.

**Additional info**
I apologise to whoever added this. My intention is not to be mean! 😄

**Comment by Thomas101** [comment]
Yep, it can be a real pain when there are more than 2 or 3 services in an account. We originally added it to indicate that you're partway down the page, but I think we've outgrown that. I'll have a look to either disable it or do something a bit better

**Comment by pauby** [comment]
This is still a thing in version 10 - come on guys, it's nothing but eye candy. I want to jump to settings that I ask for not have something 'scroll magically before my eyes'. At least give us a way to turn it off.

**Comment by pauby** [comment]
Actually, this isn't fixed at all. 

If you right-click the tab you want and click Settings it jumps immediately to the tab in the setting pages. However if you click Wavebox -> Settings -> Groups and Apps and then select an app on the left hand side it still scrolls to it.

I can't provide a screenshot of this due to all of my tabs, accounts etc. being shown.
